### PR TITLE
Create Jooheon_Yoo.markdown

### PR DESCRIPTION
**Pull request by jyoo1042**

### CODE REVIEW

File: _people/Jooheon_Yoo.markdown [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+name: Jooheon Yoo
+lastname: Yoo
+group: graduate_student
```
Comment (duetosymmetry) [inline _people/Jooheon_Yoo.markdown]:
```suggestion
group: graduate_students
```

**Comment by duetosymmetry** [comment]
Besides the group name (`graduate_students`), a more narrative biography would be nice (but won't hold up the merge)

File: _people/Jooheon_Yoo.markdown (hunk 1 above)
Comment (jyoo1042) [reply _people/Jooheon_Yoo.markdown]: I just fixed it. Thanks!

**Comment by jyoo1042** [comment]
I just fixed the group name and added a short bio.

**Pull request merged**